modules/gitops-release-notes-1-4-5.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-5_{context}"]
+= Release notes for {gitops-title} 1.4.5
+
+[role="_abstract"]
+{gitops-title} 1.4.5 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="fixed-issues-1-4-5_{context}"]
+== Fixed issues
+
+The following issue has been resolved in the current release:
+
+
+[WARNING]
+====
+Red Hat strongly recommends to directly upgrade to {gitops-title} v1.4.5 from {gitops-title} v1.4.3. Do not use {gitops-title} v1.4.4 in the production environment. Some major issues that affected {gitops-title} v1.4.4 were fixed in {gitops-title} 1.4.5. 
+====
+
+* Before this update, multiple pods were stuck in the `ErrImagePullBackOff` state. The following error message was shown:
```

Comment:
The way it's stated here, it sounds like multiple pods were always stuck in this error state. Is this true? Or did it only happen sometimes under specific circumstances? If the latter, then re-word this sentence for accuracy.

Reply:
@iam-veeramalla please suggest

Reply:
it happens every time with v1.4.4

Reply:
instead of multiple pods, you can say `argocd pods`